### PR TITLE
WL-1209,WL-1223 : Display m.ox URLs alongside regular URLs in poll tool

### PR DESCRIPTION
**Pull request by nicholaswilson100**
I've applied the 2 patches and the jquery files that Steve Swinsburg attached to the JIRAs

### CODE REVIEW

File: tool/src/webapp/js/polls.js [hunk 1]
```diff
@@ -19,55 +19,82 @@ $(document).ready(function() {
...
+            type: "GET",
+            cache: true,
+            dataType: "text",
+            timeout: 5000,
+            success: function(data) {
+                //set shortenedUrl into link
+                $("#dialog-mox-url").html(data);
+
+                //also set img tag for QR code
+                //var imgUrl = "http://chart.apis.google.com/chart?chs=250x250&cht=qr&chl=" + data;
```
Comment (ghost) [inline tool/src/webapp/js/polls.js]: I wouldn't worry about commenting out, we have version control for that. Just delete the line.

File: tool/src/webapp/templates/votePolls.html [hunk 2]
```diff
@@ -7,9 +7,9 @@
 <link href="../css/poll_base.css" type="text/css" rel="stylesheet" media="all"/>
 <link href="/library/js/jquery/ui/1.8.4/css/ui-lightness/jquery-ui-1.8.4.full.css" type="text/css" rel="stylesheet" media="all"/>
 <script  rsf:id="scr=portal-matter" type="text/javascript" language="JavaScript" src="/library/js/headscripts.js"></script>
-<script language="javascript" type="text/javascript" src="/library/js/jquery/ui/1.8.4/jquery-ui-1.8.4.full.min.js"></script>
-<script type="text/javascript" src="/library/js/jquery/jquery-1.9.1.min.js"></script>
+<script language="javascript" type="text/javascript" src="/library/js/jquery/1.4.2/jquery-1.4.2.min.js"></script>
```
Comment (ghost) [inline tool/src/webapp/templates/votePolls.html]: Do we not use jquery 1.9.1 locally?

**Comment by nicholaswilson100** [comment]
Cool thanks I've deleted some comments. 

We do use 1.9.1 locally but the QR pop up only works with 1.4.2.

**Comment by ghost** [comment]
That really sucks, I imagine it shouldn't be too difficult to get the code to work with 1.9.1. I normally find the difference is in using `prop` / `attr`.

Is it just the `polls.js` file that has QR logic?

**Comment by nicholaswilson100** [comment]
Yes poll.js is where the QR magic happens. I can have a look at making it work with 1.9.1

**Comment by ghost** [comment]
Ok sweet, thanks :+1:

**Comment by ghost** [comment]
This is also for WL-3412.

**Comment by nicholaswilson100** [comment]
Cool it's working now with JQuery 1.9.1 rather than 1.4. It needed a later version of jquery-ui.{versionId}.full.min.js

**Pull request merged**